app/src/main/java/ca/fuwafuwa/kaku/Windows/CaptureWindow.java

```diff
@@ -56,7 +57,13 @@ public boolean onTouch(MotionEvent e) {
     @Override
     public boolean onDoubleTap(MotionEvent e) {
         tessStarted = true;
-        mTessRunnable.runTess(new BoxParams(params.x, params.y + getStatusBarHeight(), params.width, params.height));
+
+        int[] viewPos = new int[2];
+        mWindowBox.getLocationOnScreen(viewPos);
+
+        // TODO: Replace the first 1 with R.drawable.border_translucent.StrokeWidth when I figure out how
+        int offset = KakuTools.dpToPx(this.context, 1)+1;
+        mTessRunnable.runTess(new BoxParams(viewPos[0]+offset, viewPos[1]+offset, params.width-(2*offset), params.height-(2*offset)));
```

Comment:
My thought process was to keep the offset calculation in the class that has the offset. That way the tesseract program does not need to worry about adjusting the box and whatnot, it just runs on the information provided.

Would it be alright to have the CaptureWindow class calculate the border width at construction and have a public getter? This way at least the variable is still calculated in Capture window, (Which RunTess has access to)